package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "the-todo-list",
+  "version": "1.0.0",
+  "description": "In this project, we have build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.",
+  "private": "true",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
```

Comment:
- [ ] **When I run this script, the HTML document is not loaded. Please fix this**

Upon running `npm run start` I am getting a black document with no elements. For some reason, your HTML document is not loaded. Please fix this.